projects/aca-content/src/lib/components/sidenav/directives/action.directive.ts

```diff
@@ -36,7 +37,7 @@ import { AppStore } from '@alfresco/aca-shared/store';
 export class ActionDirective {
   @Input() action;
 
-  @Output() actionClicked = new EventEmitter<void>();
+  @Output() actionClicked = new EventEmitter<NavBarLinkRef>();
```

Comment:
If you remove actionClicked output from expandMenu then probably here you can remove this as well as it will be unused then